server/src/main/java/umm3601/Main.java

```diff
@@ -42,16 +43,21 @@ static String getEnvOrDefault(String envName, String defaultValue) {
   }
 
   /**
-   * Get the controllers for the server. You'll add your own controllers
-   * here as you create them.
+   * Get the implementations of `AddsRoutes` used for the server.
+   *
+   * These will presumably be one or more controllers, each of which
+   * implements the `AddsRoutes` interface. You'll add your own controllers
+   * in `getRouteAdders` as you create them.
    *
    * @param database The MongoDB database object used by the controllers
    *               to access the database.
-   * @return An array of controllers for the server.
+   * @return An array of implementations of `AddRoutes` for the server.
```

Comment:
I think the use of "AddRoutes" here is a bit confusing. This will be an array of controllers that implement the interface you renamed... ("AddsRoutes"... which is mentioned here as "AddRoutes" and in "getRouteAdders"). I feel like there is a little inconsistency that should be fixed. line 27 kind of implies it will be RouteAdders.